features/unattached_commands.feature

```diff
@@ -0,0 +1,63 @@
+Feature: Command behaviour when unattached
+
+    Scenario: Unattached detach in a trusty lxd container
+        Given a trusty lxd container
+          And ubuntu-advantage-client is installed
+        When I run `ua detach`
```

Comment:
should we be verbose of root versus non-root usage or only call out when a non-root user does something and always assume root is doing things?

e.g. root runs or sudo runs
e.g. non-root runs or user runs

Reply:
Yes, I think that makes sense.

Reply:
Done, as:

```gherkin
When I run `ua status` as non-root
```